gen/build_deploy/aws.py

```diff
@@ -128,15 +128,15 @@ def validate_provider(provider):
 
 region_to_ami_map = {
     'ap-northeast-1': {
-        'coreos': 'ami-93f2baf4',
-        'stable': 'ami-93f2baf4',
+        'coreos': 'ami-ab20efd4',
+        'stable': 'ami-ab20efd4',
         'el7': 'ami-965345f8',
         'el7prereq': 'ami-f1aee697',
         'natami': 'ami-55c29e54'
     },
     'ap-southeast-1': {
-        'coreos': 'ami-aacc7dc9',
-        'stable': 'ami-aacc7dc9',
+        'coreos': 'ami-961ac5f4',
```

Comment:
This AMI is for `ap-southeast-2`, the `ap-southeast-1` should be `ami-3cded940`.
I think you accidentally skipped updating `ap-southeast-2` (line 144), which should be updated to `ami-961ac5f4`

Reply:
You're absolutely right! Good catch. Fixed.